docs/UserGuide.md

```diff
@@ -30,152 +34,130 @@ FABook is a **desktop app for managing contacts, optimized for a financial advis
 
    * **`exit`** : Exits the app.
 
-1. Refer to the [Features](#features) below for details of each command.
+6. Refer to the [Features](#features) below for details of each command.
+7. Refer to the [Command Summary](#command-summary) for a quick summary of all commands.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Features
 
 <div markdown="block" class="alert alert-info">
 
-**:information_source: Notes about the command format:**<br>
+**:information_source: Overview of commands:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `create n/NAME`, `NAME` is a parameter which can be used as `create n/John Doe`.
+* Words in `UPPER_CASE` are the information supplied by you.
+  e.g `NAME` is a parameter in `create n/NAME`, where you can input `create n/John Doe`. 
+  <br>
 
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [a/HOME_ADDRESS]` can be used as `n/John Doe a/Blk 30 Geylang Street 29` or as `n/John Doe`.
+* Parameters in `[]` are optional.
+  e.g `HOME_ADDRESS` is an optional parameter in `create n/NAME [a/HOME_ADDRESS]`, where you can input `create n/John Doe a/Blk 30 Geylang Street 29` or just `create n/John Doe`.
+  <br>
 
-* Items with `…`​ after them can be used multiple times.<br>
-  e.g. `NAME…​` can be used as `Jon`, `Jon Jack` etc.
+* Items with `…`​ after them can be used multiple times.
+  e.g `NAME…​` can be used as `Jon`, `Jon Jack` etc.
+  <br>
 
 * `INDEX` represents the index of a specific person in the FABook
-  e.g. `delete INDEX` can be used as `delete 2`.
+  e.g You can use the command `delete INDEX` as `delete 2`.
+<br>
 
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/HP_NUMBER`, `p/HP_NUMBER n/NAME` is also acceptable.
+* Parameters can be in **any order**.
+  e.g You can input either `create n/NAME p/HP_NUMBER` or ` create p/HP_NUMBER n/NAME`.
+  <br>
 
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+* Only the last occurrence of a parameter that is expected once will be taken **if you specify it more than once.**
+  e.g Given `p/12341234 p/56785678`, only `p/56785678` will be taken.
+  <br>
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit`,`clear`, 'redo' and 'undo') will be ignored.<br>
+* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit`,`clear`, 'redo' and 'undo') will be ignored.
```

Comment:
My mistake, I think didnt put redo and undo in the correct format, should be using `redo` instead of 'redo'